### ISSUE TITLE
2D->3D

### ISSUE THREAD

**Issue by patricio-astudillo** [kenshohara/3D-ResNets-PyTorch#112]
https://github.com/kenshohara/3D-ResNets-PyTorch/blob/f399b376ca555f0ff925d77517313164c66504f9/models/densenet.py#L193

This gives an error on my side, stating that the 2D layer expects a 2D input. Should this be a 3D BatchNorm layer instead?

**Comment by PiggyGenius** [comment]
I am wondering the same thing. Did you figure this out ?

**Comment by mijung-kim** [comment]
@patricio-astudillo In case you haven't solved it yet. I changed it to 2D to 3D because it throws out a dimension error as you mentioned. Then, the error is gone and worked well.
